tap_freshdesk/client.py

```diff
@@ -27,16 +132,27 @@ def __exit__(self, exception_type, exception_value, traceback):
         # Kill the session instance.
         self.session.close()
 
+    def set_timeout(self):
+        """
+        Set timeout value from config, if the value is passed. 
+        Else raise an exception.
+        """
+        timeout = self.config.get("timeout", DEFAULT_TIMEOUT)
```

Comment:
Add detail about `request_timeout` in the README file.

Reply:
Added detail about `request_timeout` in the README file.